controllers/managedocs_controller_test.go

```diff
@@ -18,7 +19,8 @@ import (
 var _ = Describe("ManagedOCS controller", func() {
 	// Define utility constants for object names and testing timeouts/durations and intervals.
 	const (
-		ManagedOCSName = "test-managedocs"
+		ManagedOCSName   = "test-managedocs"
+		ConfigSecretName = "test-secret"
```

Comment:
This const should be defined on the test_suite to be shared with all tests. 
Also, the name is misleading, let's call it something like  `AddOnParamsSecretName`

Reply:
In that case should that be done for the other names as well (namespace, managedOcs etc)

Reply:
If we need to use a const both on more than one test file or on a test file and the test_suite it should be defined in a single location